tests/test_yask.py

```diff
@@ -17,23 +19,27 @@
                                 reason="'yask' wasn't selected as backend on startup")
 
 
+@skipif_backend(['ops'])
```

Comment:
I **think** you don't need `skipif_backend('ops')` in `yask.py` because of [this](https://github.com/opesci/devito/pull/637/files#diff-c8cec68f633682b26c422e4467cb8e14R18)

Reply:
ok!